### PR TITLE
Even More Tied Appliances

### PR DESCRIPTION
**Pull request by Holli-Git**
#### Summary
Content "Even more tied appliances"

#### Purpose of change

There was more stuff that could be tied.

#### Describe the solution

Adds it, also removes the EXTRA_DRAG flag as it didn't seem give extra drag or lower fuel consumption in this context, and gave an annoying descriptor in the install menu.

#### Testing

Lo and behold!
image

#### Additional context

As evidence the EXTRA_DRAG flag doesn't do anything for air drag, here's a vehicle with 8 jointers that have the EXTRA_DRAG flag
image
Here's one without EXTRA_DRAG flag
image

### CODE REVIEW

**Review by Maleclypse — CHANGES_REQUESTED** [review]
```
  if( vp.part().enabled && vpi.has_flag( "EXTRA_DRAG" ) ) {
            extra_drag += vpi.power;
        }
        if( vpi.has_flag( "EXTRA_DRAG" ) && ( vpi.has_flag( "WIND_TURBINE" ) ||
                                              vpi.has_flag( "WATER_WHEEL" ) ) ) {
            extra_drag += vpi.power;
```
Extra Drag is in code as doing "something" so it's either broken in code or displaying without the changes it is applying.    Two very different states neither of which is solved by removing EXTRA_DRAG flags.   Better to put them back and make an issue explaining that it's either not applying changes despite math in code or not displaying those changes to the player.

**Comment by Holli-Git** [comment]
> ```
>   if( vp.part().enabled && vpi.has_flag( "EXTRA_DRAG" ) ) {
>             extra_drag += vpi.power;
>         }
>         if( vpi.has_flag( "EXTRA_DRAG" ) && ( vpi.has_flag( "WIND_TURBINE" ) ||
>                                               vpi.has_flag( "WATER_WHEEL" ) ) ) {
>             extra_drag += vpi.power;
> ```
> 
> Extra Drag is in code as doing "something" so it's either broken in code or displaying without the changes it is applying. Two very different states neither of which is solved by removing EXTRA_DRAG flags. Better to put them back and make an issue explaining that it's either not applying changes despite math in code or not displaying those changes to the player.

Alright, though I am worried the message about "When turned on, it will stop the vehicle unless it has a strong engine" confusing people

**Comment by Maleclypse** [comment]
OK so I went and spent some time learning about what adds drag to a vehicle https://www.autospeed.com/cms/a_113227/article.html is a solid breakdown.   And the wake of a vehicle which is reduced by the modern shape of a car is by far the largest producer of drag on a vehicle making up 70-80% of total drag.  And as shown by the picture of the wagon/hatchback, you can see how changing the shape of the roof has a massive impact on drag, and tied objects change the shape of the roof in comparison to normal vehicle design.  So I'm going to put the extra drag flags back on then let the tests run on this and hopefully they all pass and it merges and I can create an issue for "whether or not extra drag is working and if not why" if it doesn't already exist.  :)  Thanks for putting up with me on this.

File: data/json/vehicleparts/stowed_parts.json [hunk 1]
```diff
@@ -13,15 +13,14 @@
     "location": "on_roof",
     "requirements": { "install": { "time": "10 m", "using": [ [ "rope_natural", 1 ] ] }, "removal": { "time": "5 m" } },
     "breaks_into": [ { "item": "apartment_fridge" }, { "item": "rope_30" } ],
```
Comment (Maleclypse) [inline data/json/vehicleparts/stowed_parts.json:15]:
```suggestion
    "breaks_into": [ { "item": "apartment_fridge" }, { "item": "rope_30" } ],
    "flags": [ "EXTRA_DRAG" ],
```

File: data/json/vehicleparts/stowed_parts.json [hunk 2]
```diff
@@ -30,15 +29,14 @@
     "location": "on_roof",
     "requirements": { "install": { "time": "10 m", "using": [ [ "rope_natural", 1 ] ] }, "removal": { "time": "5 m" } },
     "breaks_into": [ { "item": "minifridge" }, { "item": "rope_30" } ],
```
Comment (Maleclypse) [inline data/json/vehicleparts/stowed_parts.json:31]:
```suggestion
    "breaks_into": [ { "item": "minifridge" }, { "item": "rope_30" } ],
    "flags": [ "EXTRA_DRAG" ],
```

File: data/json/vehicleparts/stowed_parts.json [hunk 3]
```diff
@@ -47,15 +45,14 @@
...
```
Comment (Maleclypse) [inline data/json/vehicleparts/stowed_parts.json:47]:
```suggestion
    "breaks_into": [ { "item": "minifridge" }, { "item": "rope_30" } ],
    "flags": [ "EXTRA_DRAG" ],
```

File: data/json/vehicleparts/stowed_parts.json [hunk 4]
```diff
@@ -64,15 +61,14 @@
     "location": "on_roof",
     "requirements": { "install": { "time": "10 m", "using": [ [ "rope_natural", 1 ] ] }, "removal": { "time": "5 m" } },
     "breaks_into": [ { "item": "glass_fridge" }, { "item": "rope_30" } ],
```
Comment (Maleclypse) [inline data/json/vehicleparts/stowed_parts.json:63]:
```suggestion
    "breaks_into": [ { "item": "glass_fridge" }, { "item": "rope_30" } ],
    "flags": [ "EXTRA_DRAG" ],
```

File: data/json/vehicleparts/stowed_parts.json [hunk 5]
```diff
@@ -81,15 +77,14 @@
     "location": "on_roof",
     "requirements": { "install": { "time": "10 m", "using": [ [ "rope_natural", 1 ] ] }, "removal": { "time": "5 m" } },
     "breaks_into": [ { "item": "glass_fridge_double" }, { "item": "rope_30" } ],
```
Comment (Maleclypse) [inline data/json/vehicleparts/stowed_parts.json:79]:
```suggestion
    "breaks_into": [ { "item": "glass_fridge_double" }, { "item": "rope_30" } ],
    "flags": [ "EXTRA_DRAG" ],
```

File: data/json/vehicleparts/stowed_parts.json [hunk 6]
```diff
@@ -98,15 +93,14 @@
     "location": "on_roof",
     "requirements": { "install": { "time": "15 m", "using": [ [ "rope_natural", 2 ] ] }, "removal": { "time": "10 m" } },
     "breaks_into": [ { "item": "heavy_duty_fridge" }, { "item": "rope_30", "count": [ 2, 2 ] } ],
```
Comment (Maleclypse) [inline data/json/vehicleparts/stowed_parts.json:95]:
```suggestion
    "breaks_into": [ { "item": "heavy_duty_fridge" }, { "item": "rope_30", "count": [ 2, 2 ] } ],
    "flags": [ "EXTRA_DRAG" ],
```

File: data/json/vehicleparts/stowed_parts.json [hunk 7]
```diff
@@ -115,15 +109,14 @@
     "location": "on_roof",
     "requirements": { "install": { "time": "10 m", "using": [ [ "rope_natural", 1 ] ] }, "removal": { "time": "5 m" } },
     "breaks_into": [ { "item": "display_fridge" }, { "item": "rope_30" } ],
```
Comment (Maleclypse) [inline data/json/vehicleparts/stowed_parts.json:111]:
```suggestion
    "breaks_into": [ { "item": "display_fridge" }, { "item": "rope_30" } ],
    "flags": [ "EXTRA_DRAG" ],
```

File: data/json/vehicleparts/stowed_parts.json [hunk 8]
```diff
@@ -132,15 +125,14 @@
     "location": "on_roof",
     "requirements": { "install": { "time": "10 m", "using": [ [ "rope_natural", 1 ] ] }, "removal": { "time": "5 m" } },
     "breaks_into": [ { "item": "apartment_freezer" }, { "item": "rope_30" } ],
```
Comment (Maleclypse) [inline data/json/vehicleparts/stowed_parts.json:127]:
```suggestion
    "breaks_into": [ { "item": "apartment_freezer" }, { "item": "rope_30" } ],
    "flags": [ "EXTRA_DRAG" ],
```

File: data/json/vehicleparts/stowed_parts.json [hunk 9]
```diff
@@ -149,15 +141,14 @@
     "location": "on_roof",
     "requirements": { "install": { "time": "10 m", "using": [ [ "rope_natural", 1 ] ] }, "removal": { "time": "5 m" } },
     "breaks_into": [ { "item": "minifreezer" }, { "item": "rope_30" } ],
```
Comment (Maleclypse) [inline data/json/vehicleparts/stowed_parts.json:143]:
```suggestion
    "breaks_into": [ { "item": "minifreezer" }, { "item": "rope_30" } ],
    "flags": [ "EXTRA_DRAG" ],
```

File: data/json/vehicleparts/stowed_parts.json [hunk 10]
```diff
@@ -166,15 +157,14 @@
     "location": "on_roof",
     "requirements": { "install": { "time": "10 m", "using": [ [ "rope_natural", 1 ] ] }, "removal": { "time": "5 m" } },
     "breaks_into": [ { "item": "chest_minifreezer" }, { "item": "rope_30" } ],
```
Comment (Maleclypse) [inline data/json/vehicleparts/stowed_parts.json:159]:
```suggestion
    "breaks_into": [ { "item": "chest_minifreezer" }, { "item": "rope_30" } ],
    "flags": [ "EXTRA_DRAG" ],
```

File: data/json/vehicleparts/stowed_parts.json [hunk 11]
```diff
@@ -183,15 +173,14 @@
     "location": "on_roof",
     "requirements": { "install": { "time": "10 m", "using": [ [ "rope_natural", 1 ] ] }, "removal": { "time": "5 m" } },
     "breaks_into": [ { "item": "big_portable_freezer" }, { "item": "rope_30" } ],
```
Comment (Maleclypse) [inline data/json/vehicleparts/stowed_parts.json:175]:
```suggestion
    "breaks_into": [ { "item": "big_portable_freezer" }, { "item": "rope_30" } ],
    "flags": [ "EXTRA_DRAG" ],
```

File: data/json/vehicleparts/stowed_parts.json [hunk 12]
```diff
@@ -200,15 +189,14 @@
     "location": "on_roof",
     "requirements": { "install": { "time": "10 m", "using": [ [ "rope_natural", 1 ] ] }, "removal": { "time": "5 m" } },
     "breaks_into": [ { "item": "chest_freezer" }, { "item": "rope_30" } ],
```
Comment (Maleclypse) [inline data/json/vehicleparts/stowed_parts.json:191]:
```suggestion
    "breaks_into": [ { "item": "chest_freezer" }, { "item": "rope_30" } ],
    "flags": [ "EXTRA_DRAG" ],
```

File: data/json/vehicleparts/stowed_parts.json [hunk 13]
```diff
@@ -217,15 +205,14 @@
     "location": "on_roof",
     "requirements": { "install": { "time": "15 m", "using": [ [ "rope_natural", 2 ] ] }, "removal": { "time": "10 m" } },
     "breaks_into": [ { "item": "heavy_duty_freezer" }, { "item": "rope_30", "count": [ 2, 2 ] } ],
```
Comment (Maleclypse) [inline data/json/vehicleparts/stowed_parts.json:207]:
```suggestion
    "breaks_into": [ { "item": "heavy_duty_freezer" }, { "item": "rope_30", "count": [ 2, 2 ] } ],
    "flags": [ "EXTRA_DRAG" ],
```

File: data/json/vehicleparts/stowed_parts.json [hunk 14]
```diff
@@ -234,15 +221,14 @@
     "location": "on_roof",
     "requirements": { "install": { "time": "10 m", "using": [ [ "rope_natural", 1 ] ] }, "removal": { "time": "5 m" } },
     "breaks_into": [ { "item": "glass_freezer" }, { "item": "rope_30" } ],
```
Comment (Maleclypse) [inline data/json/vehicleparts/stowed_parts.json:223]:
```suggestion
    "breaks_into": [ { "item": "glass_freezer" }, { "item": "rope_30" } ],
    "flags": [ "EXTRA_DRAG" ],
```

File: data/json/vehicleparts/stowed_parts.json [hunk 15]
```diff
@@ -251,7 +237,212 @@
     "location": "on_roof",
     "requirements": { "install": { "time": "10 m", "using": [ [ "rope_natural", 1 ] ] }, "removal": { "time": "5 m" } },
     "breaks_into": [ { "item": "display_freezer" }, { "item": "rope_30" } ],
```
Comment (Maleclypse) [inline data/json/vehicleparts/stowed_parts.json:239]:
```suggestion
    "breaks_into": [ { "item": "display_freezer" }, { "item": "rope_30" } ],
    "flags": [ "EXTRA_DRAG" ],
```

File: data/json/vehicleparts/stowed_parts.json [hunk 16]
```diff
@@ -251,7 +237,212 @@
...
+    "categories": [ "other" ],
+    "color": "dark_gray_white",
+    "damage_modifier": 5,
+    "durability": 20,
+    "//": "This is used to simulate a rope being untied/broken and falling off",
+    "description": "An oven, secured to the roof for transportation.  Not functional, but easier than carrying it with your hands.",
+    "item": "oven",
+    "location": "on_roof",
+    "requirements": { "install": { "time": "10 m", "using": [ [ "rope_natural", 1 ] ] }, "removal": { "time": "5 m" } },
+    "breaks_into": [ { "item": "oven" }, { "item": "rope_30" } ],
```
Comment (Maleclypse) [inline data/json/vehicleparts/stowed_parts.json:255]:
```suggestion
    "breaks_into": [ { "item": "oven" }, { "item": "rope_30" } ],
    "flags": [ "EXTRA_DRAG" ],
```

File: data/json/vehicleparts/stowed_parts.json [hunk 17]
```diff
@@ -251,7 +237,212 @@
...
+    "categories": [ "other" ],
+    "color": "red",
+    "damage_modifier": 5,
+    "durability": 20,
+    "//": "This is used to simulate a rope being untied/broken and falling off",
+    "description": "An arcade machine, secured to the roof for transportation.  Not functional, but easier than carrying it with your hands.",
+    "item": "arcade_machine",
+    "location": "on_roof",
+    "requirements": { "install": { "time": "10 m", "using": [ [ "rope_natural", 1 ] ] }, "removal": { "time": "5 m" } },
+    "breaks_into": [ { "item": "arcade_machine" }, { "item": "rope_30" } ],
```
Comment (Maleclypse) [inline data/json/vehicleparts/stowed_parts.json:271]:
```suggestion
    "breaks_into": [ { "item": "arcade_machine" }, { "item": "rope_30" } ],
    "flags": [ "EXTRA_DRAG" ],
```

File: data/json/vehicleparts/stowed_parts.json [hunk 18]
```diff
@@ -251,7 +237,212 @@
...
+    "categories": [ "other" ],
+    "color": "yellow_red",
+    "damage_modifier": 5,
+    "durability": 20,
+    "//": "This is used to simulate a rope being untied/broken and falling off",
+    "description": "A drill press, secured to the roof for transportation.  Not functional, but easier than carrying it with your hands.",
+    "item": "drill_press",
+    "location": "on_roof",
+    "requirements": { "install": { "time": "10 m", "using": [ [ "rope_natural", 1 ] ] }, "removal": { "time": "5 m" } },
+    "breaks_into": [ { "item": "drill_press" }, { "item": "rope_30" } ],
```
Comment (Maleclypse) [inline data/json/vehicleparts/stowed_parts.json:287]:
```suggestion
    "breaks_into": [ { "item": "drill_press" }, { "item": "rope_30" } ],
    "flags": [ "EXTRA_DRAG" ],
```

File: data/json/vehicleparts/stowed_parts.json [hunk 19]
```diff
@@ -251,7 +237,212 @@
...
+    "categories": [ "other" ],
+    "color": "yellow_red",
+    "damage_modifier": 5,
+    "durability": 20,
+    "//": "This is used to simulate a rope being untied/broken and falling off",
+    "description": "A table saw, secured to the roof for transportation.  Not functional, but easier than carrying it with your hands.",
+    "item": "tablesaw",
+    "location": "on_roof",
+    "requirements": { "install": { "time": "10 m", "using": [ [ "rope_natural", 1 ] ] }, "removal": { "time": "5 m" } },
+    "breaks_into": [ { "item": "tablesaw" }, { "item": "rope_30" } ],
```
Comment (Maleclypse) [inline data/json/vehicleparts/stowed_parts.json:303]:
```suggestion
    "breaks_into": [ { "item": "tablesaw" }, { "item": "rope_30" } ],
    "flags": [ "EXTRA_DRAG" ],
```

File: data/json/vehicleparts/stowed_parts.json [hunk 20]
```diff
@@ -251,7 +237,212 @@
...
+    "categories": [ "other" ],
+    "color": "yellow_cyan",
+    "damage_modifier": 5,
+    "durability": 20,
+    "//": "This is used to simulate a rope being untied/broken and falling off",
+    "description": "A mitre, secured to the roof for transportation.  Not functional, but easier than carrying it with your hands.",
+    "item": "mitresaw",
+    "location": "on_roof",
+    "requirements": { "install": { "time": "10 m", "using": [ [ "rope_natural", 1 ] ] }, "removal": { "time": "5 m" } },
+    "breaks_into": [ { "item": "mitresaw" }, { "item": "rope_30" } ],
```
Comment (Maleclypse) [inline data/json/vehicleparts/stowed_parts.json:319]:
```suggestion
    "breaks_into": [ { "item": "mitresaw" }, { "item": "rope_30" } ],
    "flags": [ "EXTRA_DRAG" ],
```

File: data/json/vehicleparts/stowed_parts.json [hunk 21]
```diff
@@ -251,7 +237,212 @@
...
+    "categories": [ "other" ],
+    "color": "yellow_cyan",
+    "damage_modifier": 5,
+    "durability": 20,
+    "//": "This is used to simulate a rope being untied/broken and falling off",
+    "description": "A band saw, secured to the roof for transportation.  Not functional, but easier than carrying it with your hands.",
+    "item": "bandsaw",
+    "location": "on_roof",
+    "requirements": { "install": { "time": "10 m", "using": [ [ "rope_natural", 1 ] ] }, "removal": { "time": "5 m" } },
+    "breaks_into": [ { "item": "bandsaw" }, { "item": "rope_30" } ],
```
Comment (Maleclypse) [inline data/json/vehicleparts/stowed_parts.json:335]:
```suggestion
    "breaks_into": [ { "item": "bandsaw" }, { "item": "rope_30" } ],
    "flags": [ "EXTRA_DRAG" ],
```

File: data/json/vehicleparts/stowed_parts.json [hunk 22]
```diff
@@ -251,7 +237,212 @@
...
+    "categories": [ "other" ],
+    "color": "yellow_green",
+    "damage_modifier": 5,
+    "durability": 20,
+    "//": "This is used to simulate a rope being untied/broken and falling off",
+    "description": "A router, secured to the roof for transportation.  Not functional, but easier than carrying it with your hands.",
+    "item": "router",
+    "location": "on_roof",
+    "requirements": { "install": { "time": "10 m", "using": [ [ "rope_natural", 1 ] ] }, "removal": { "time": "5 m" } },
+    "breaks_into": [ { "item": "router" }, { "item": "rope_30" } ],
```
Comment (Maleclypse) [inline data/json/vehicleparts/stowed_parts.json:351]:
```suggestion
    "breaks_into": [ { "item": "router" }, { "item": "rope_30" } ],
    "flags": [ "EXTRA_DRAG" ],
```

File: data/json/vehicleparts/stowed_parts.json [hunk 23]
```diff
@@ -251,7 +237,212 @@
...
+    "categories": [ "other" ],
+    "color": "yellow_white",
+    "damage_modifier": 5,
+    "durability": 20,
+    "//": "This is used to simulate a rope being untied/broken and falling off",
+    "description": "A planer, secured to the roof for transportation.  Not functional, but easier than carrying it with your hands.",
+    "item": "planer",
+    "location": "on_roof",
+    "requirements": { "install": { "time": "10 m", "using": [ [ "rope_natural", 1 ] ] }, "removal": { "time": "5 m" } },
+    "breaks_into": [ { "item": "planer" }, { "item": "rope_30" } ],
```
Comment (Maleclypse) [inline data/json/vehicleparts/stowed_parts.json:367]:
```suggestion
    "breaks_into": [ { "item": "planer" }, { "item": "rope_30" } ],
    "flags": [ "EXTRA_DRAG" ],
```

File: data/json/vehicleparts/stowed_parts.json [hunk 24]
```diff
@@ -251,7 +237,212 @@
...
+    "categories": [ "other" ],
+    "color": "yellow_magenta",
+    "damage_modifier": 5,
+    "durability": 20,
+    "//": "This is used to simulate a rope being untied/broken and falling off",
+    "description": "A jointer, secured to the roof for transportation.  Not functional, but easier than carrying it with your hands.",
+    "item": "jointer",
+    "location": "on_roof",
+    "requirements": { "install": { "time": "10 m", "using": [ [ "rope_natural", 1 ] ] }, "removal": { "time": "5 m" } },
+    "breaks_into": [ { "item": "jointer" }, { "item": "rope_30" } ],
```
Comment (Maleclypse) [inline data/json/vehicleparts/stowed_parts.json:383]:
```suggestion
    "breaks_into": [ { "item": "jointer" }, { "item": "rope_30" } ],
    "flags": [ "EXTRA_DRAG" ],
```

File: data/json/vehicleparts/stowed_parts.json [hunk 25]
```diff
@@ -251,7 +237,212 @@
...
+    "categories": [ "other" ],
+    "color": "black_red",
+    "damage_modifier": 5,
+    "durability": 20,
+    "//": "This is used to simulate a rope being untied/broken and falling off",
+    "description": "A hydraulic press, secured to the roof for transportation.  Not functional, but easier than carrying it with your hands.",
+    "item": "hydraulic_press",
+    "location": "on_roof",
+    "requirements": { "install": { "time": "10 m", "using": [ [ "rope_natural", 1 ] ] }, "removal": { "time": "5 m" } },
+    "breaks_into": [ { "item": "hydraulic_press" }, { "item": "rope_30" } ],
```
Comment (Maleclypse) [inline data/json/vehicleparts/stowed_parts.json:399]:
```suggestion
    "breaks_into": [ { "item": "hydraulic_press" }, { "item": "rope_30" } ],
    "flags": [ "EXTRA_DRAG" ],
```

File: data/json/vehicleparts/stowed_parts.json [hunk 26]
```diff
@@ -251,7 +237,212 @@
...
+    "categories": [ "other" ],
+    "color": "black_yellow",
+    "damage_modifier": 5,
+    "durability": 20,
+    "//": "This is used to simulate a rope being untied/broken and falling off",
+    "description": "An air compressor, secured to the roof for transportation.  Not functional, but easier than carrying it with your hands.",
+    "item": "air_compressor",
+    "location": "on_roof",
+    "requirements": { "install": { "time": "10 m", "using": [ [ "rope_natural", 1 ] ] }, "removal": { "time": "5 m" } },
+    "breaks_into": [ { "item": "air_compressor" }, { "item": "rope_30" } ],
```
Comment (Maleclypse) [inline data/json/vehicleparts/stowed_parts.json:415]:
```suggestion
    "breaks_into": [ { "item": "air_compressor" }, { "item": "rope_30" } ],
    "flags": [ "EXTRA_DRAG" ],
```

File: data/json/vehicleparts/stowed_parts.json [hunk 27]
```diff
@@ -251,7 +237,212 @@
...
+    "name": { "str": "tied-down oxygen concentrator" },
+    "categories": [ "other" ],
+    "damage_modifier": 5,
+    "durability": 20,
+    "//": "This is used to simulate a rope being untied/broken and falling off",
+    "description": "An oxygen concentrator, secured to the roof for transportation.  Not functional, but easier than carrying it with your hands.",
+    "item": "oxygen_concentrator",
+    "location": "on_roof",
+    "requirements": { "install": { "time": "10 m", "using": [ [ "rope_natural", 1 ] ] }, "removal": { "time": "5 m" } },
+    "breaks_into": [ { "item": "oxygen_concentrator" }, { "item": "rope_30" } ],
```
Comment (Maleclypse) [inline data/json/vehicleparts/stowed_parts.json:430]:
```suggestion
    "breaks_into": [ { "item": "oxygen_concentrator" }, { "item": "rope_30" } ],
    "flags": [ "EXTRA_DRAG" ],
```

File: data/json/vehicleparts/stowed_parts.json [hunk 28]
```diff
@@ -251,7 +237,212 @@
...
+    "name": { "str": "tied-down high pressure compressor unit" },
+    "categories": [ "other" ],
+    "damage_modifier": 5,
+    "durability": 20,
+    "//": "This is used to simulate a rope being untied/broken and falling off",
+    "description": "An oxygen concentrator, secured to the roof for transportation.  Not functional, but easier than carrying it with your hands.",
+    "item": "hd_compressor_unit",
+    "location": "on_roof",
+    "requirements": { "install": { "time": "10 m", "using": [ [ "rope_natural", 1 ] ] }, "removal": { "time": "5 m" } },
+    "breaks_into": [ { "item": "hd_compressor_unit" }, { "item": "rope_30" } ],
```
Comment (Maleclypse) [inline data/json/vehicleparts/stowed_parts.json:445]:
```suggestion
    "breaks_into": [ { "item": "hd_compressor_unit" }, { "item": "rope_30" } ],
    "flags": [ "EXTRA_DRAG" ],
```

**Review by Maleclypse — APPROVED** [review]
I was the change I wished to see in the world

**Comment by Holli-Git** [comment]
> I was the change I wished to see in the world

Ah, thank you!

**Pull request merged**